### PR TITLE
[master]Update Jarvis Profile

### PR DESCRIPTION
**Pull request by V35games**
### Changes
- Added all Jarvis Profile-related files to the new `profile` directory
- Updated profile information
- Updated master README.md

### Testing
-   [X] Tested with the provided bash script

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,89 @@
...
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Vishay Singh . Jarvis Consulting
 
+Hello there! My name is Vishay Singh and I graduated from the University of Toronto with an Honours 
+Bachelor's Degree of Science, majoring in Computer Science and French Studies. Since a young age, I've
+been fascinated by what technology can do to solve real world problems, which is exactly what I want 
+to do in the software industry. In fact, learning new things is a passion of mine, so software provides 
+the perfect avenue to continually challenge myself. In my spare time, I'm either building computers, 
+playing video games, or learning guitar.
```
Comment (jarviscanada) [inline README.md]: Lots of flavour, but need more technical details on your skills and experience. What projects have you worked on, what technologies have you used, what sorts of problems have you spent time on or like to solve, etc. This introduction should essentially include a highlights/summary of the rest of the profile.

Comment (jarviscanada) [reply README.md]: Take a look at these sample profiles for ideas (I do like the attitude and personal flair you express here though, try not to lose that)
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer

File: README.md [hunk 2]
```diff
@@ -1,8 +1,89 @@
...
+
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Python, IDEs (IntelliJ, Visual Studio Code, Eclipse, PyCharm, etc.), REST API, Docker, DAO design pattern, CRUD, JDBC, Regular Expressions, MVC architecture, Spring/SpringBoot, JUnit, Lambda Expressions, Stream API
```
Comment (jarviscanada) [inline README.md]: Some of the more generic, highly-specific, or non-technical elements don't need to be listed here. E.g. IDEs, CRUD, RegEx, Lambda/Stream, MVC. These are either generally expected or included within other things you've listed.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,89 @@
...
+databases, Maven, and deployed using Docker.
+
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/python_data_anlytics)]: Not started.
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/hadoop)]: Not started.
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/spark)]: Not started.
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/cloud_devops)]: Not started.
```
Comment (jarviscanada) [inline README.md]: remove/comment out projects that haven't been started.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,89 @@
...
+that exploit the Linux LD_PRELOAD environment variable.
+
+
+
+## Professional Experiences
+
+**Software Engineer (Bilingual / Entry Level), Jarvis (2022-present)**: Designed, implemented, tested, and deployed software projects relating to Linux/Bash scripting, 
+SQL, and Java, using Agile/Scrum methodology, Git Flow, and Docker. Attended and passed Master 
+Code Review meetings that certify acceptable code quality and requisite knowledge. Lead daily 
+stand-up meetings during the first and fifth sprints, which coordinated and tracked scrum team progress.
```
Comment (jarviscanada) [inline README.md]: Don't specify which sprints you lead. This is one of those scenarios where less is more.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,89 @@
...
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/linux_sql)]: Designed, implemented, tested, and deployed a Linux Cluster Monitoring Agent that allows businesses 
+to monitor and find server nodes that are underperforming, resulting in optimized resource allocation 
+for the entire cluster.
+
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VishaySingh/tree/master/core_java)]:
+      
+  - Twitter App: Designed, implemented, tested, and deployed a backend Java app that can CRUD tweets using Twitter's REST API. Common design patterns & frameworks were leveraged to design this app, such as MVC architecture, DAOs, and Spring Dependency Injection. Implemented using HTTP client, core Java libraries, maven, and SpringBoot. Integration and Unit testing using JUnit and Mockito, resulted in a well-tested app published on DockerHub.
```
Comment (jarviscanada) [inline README.md]: Only CRD

**Review by jarviscanada — CHANGES_REQUESTED** [review]
See comments

**Comment by V35games** [comment]
I just pushed changes that should address the comments. Please review when possible.

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**